src/openforms/submissions/templates/admin/submissions/submission/logs.html

```diff
@@ -43,11 +56,24 @@ <h4> {% trans "Logic rules" %} </h4>
                             {% for rule in log.extra_data.log_evaluated_rules %}
                                     <tr>
                                         <td>
-                                            {{ rule.source_components }}
+                                            {{ rule.rule_as_str }}
                                         </td>
                                         <td>{{ rule.trigger|yesno|capfirst }} </td>
                                         <td>
-                                            {{ rule.targeted_components }}
+                                        {% for action in rule.targeted_components %}
+                                            {% if action.key %}
+                                                <b>{{ action.step_name }}:{{ action.key }} &#40;{{ action.label }}&#41;</b><i> {{ action.type }}</i> 
```

Comment:
```suggestion
                                                <b>{{ action.step_name }}:{{ action.key }} ({{ action.label }})</b><i> {{ action.type }}</i> 
```

you don't need to use HTML escape codes for these things :)